src/Avalonia.Base/Media/DrawingGroup.cs

```diff
@@ -492,6 +494,11 @@ private void AddDrawing(Drawing newDrawing)
                     _currentDrawingGroup.Children.Add(newDrawing);
                 }
             }
+
+            public void DrawBitmap(IRef<IBitmapImpl> source, double opacity, Rect sourceRect, Rect destRect)
```

Comment:
Why do we have this if it's not implemented?

Reply:
Corrected